javascript2-week2/quizApp2.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Quiz App</title>
+    <link rel="stylesheet" type="text/css" href="quizApp2.css">
+</head>
+
+<body>
+    <div class="quiz-form">
+        <h2>Add a Quiz Question</h2>
+        <form id="quizForm">
+            <div class="form-group">
+                <label for="question">Question (max 140 characters):</label>
+                <input type="text" id="question" maxlength="140" required>
```

Comment:
This could also be a `textarea` element. If you swap the input for that user will be able to read the entire question that they are typing.